packages/inter-protocol/src/proposals/utils.js

```diff
@@ -52,6 +50,12 @@ export const reserveThenGetNames = async (nameAdmin, names) =>
     names.map(name => [name]),
   );
 
+/**
+ * @param {string} debugName
```

Comment:
Maybe `displayName`?   `debugName` seems pejorative.

Reply:
these are just jsdocs for params that were already there. The type annotation serves this PR. Changing the param name doesn't. It doesn't seem worth a drive-by either. Since you said "maybe" I take it that's not an approval requirement.